### DISCUSSION TITLE
Criei um jogo da cobrinha, só que diferente...

### DISCUSSION THREAD

**Discussion by belmirofss** [frontendbr/forum#2189] [Mostre seu trabalho]
Salve salve galera, tudo bem com vocês?

Há um tempo atrás tava com um tempo livre e queria fazer um joguinho mobile, acabei fazendo o clássico jogo da cobrinha só que com uma mecânica diferente... A cobra é controlada através dos sensores, ou seja, você controla ela como se fosse um volante.

Fiz for fun mesmo.

**Quem quiser dar uma olhada:**
https://play.google.com/store/apps/details?id=com.yabcompany.snakeshakegame

**Quem se interessar por ver o código (React & Expo):**
https://github.com/belmirofss/snake-shake-game

**Comment by peidrao** [comment]
Parabéns cara, ficou muito bom!
